packages/api/src/users/users.service.ts

```diff
@@ -1,5 +1,8 @@
 import { Injectable } from '@nestjs/common';
 
+import { User } from '@coderscamp/shared/models/user';
+
+import { GithubUserData } from '../auth/auth.model';
```

Comment:
As those are only types you can write `import type` here. It's a great way to reduce the chance of circular dependencies error in the future.